website/content/v1.3/introduction/getting-started.md

```diff
@@ -251,28 +259,58 @@ install:
 
 to reflect `vda` instead of `sda`.
 
+### Customizing Machine Configuration
+
+The generated machine configuration provides sane defaults for most cases, but machine configuration
+might be modified to fit specific needs.
+
+Some machine configuration options are available as flags for the `talosctl gen config` command,
+for example setting a specific Kubernetes version:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml --kubernetes-version 1.25.4 my-cluster https://192.168.64.15:6443
+```
+
+Other modifications are done with [machine configuration patches]({{< relref "../talos-guides/configuration/patching" >}}).
+Machine configuration patches can be applied with `talosctl gen config` command:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml --config-patch-control-plane @cni.patch my-cluster https://192.168.64.15:6443
+```
+
 #### Machine Configs as Templates
 
 Individual machines may need different settings: for instance, each may have a
 different [static IP address]({{< relref "../advanced/advanced-networking/#static-addressing" >}}).
 
-When different files are needed for machines of the same type, simply
-copy the source template (`controlplane.yaml` or `worker.yaml`) and make whatever
-modifications are needed.
+When different files are needed for machines of the same type, there are two supported flows:
 
-For instance, if you had three control plane nodes and three worker nodes, you
-may do something like this:
+1. Use the `talosctl gen config` command to generate a template, and then patch
+   the template for each machine with `talosctl machineconfig patch`.
+2. Generate each machine configuration file separately with `talosctl gen config` applying patches.
 
-```bash
-for i in $(seq 0 2); do
-  cp controlplane.yaml cp$i.yaml
-end
-for i in $(seq 0 2); do
-  cp worker.yaml w$i.yaml
-end
+For example, given a machine configuration patch which sets the static machine hostname:
+
+```yaml
+# worker1.patch
+machine:
+  network:
+    hostname: worker1
 ```
 
-Then modify each file as needed.
+Either of the following commands will generate a worker machine configuration file with the hostname set to `worker1`:
+
+```sh
```

Comment:
I would swap these examples, because they are reversed from the flows list above.

Reply:
thank you, I missed that!